src/twisted/internet/test/test_inlinecb.py

```diff
@@ -145,3 +147,181 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         self.assertIn("in erroring", f.getTraceback())
+
+
+
+class TranslateMe(Exception):
+    """
+    Sample exception type.
+    """
+
+
+
+class TranslateResult(Exception):
+    """
+    Sample exception type.
+    """
+
+
+
+class DontFail(Exception):
+    """
+    Sample exception type.
+    """
+
+    def __init__(self, actual):
+        Exception.__init__(self)
+        self.actualValue = actual
+
+
+
+class CancellationTests(TestCase):
```

Comment:
This should be [`SynchronousTestCase`](https://twistedmatrix.com/documents/18.4.0/api/twisted.trial.unittest.SynchronousTestCase.html) because none of the tests use the global reactor.